### DISCUSSION TITLE
Onnx 'InferenceSession(modelData)' while running from Windows server 2012 R2

### DISCUSSION THREAD

**Discussion by haligeorge** [onnx/onnx#5652] [Q&A]
I have developed a code in C# (.net Frame work 4.5.2) and is working from my local (Windows 10), but when I deploy or run it from the server I am getting error and not able to predict.
It is breaking in the following line
`var session = new InferenceSession(modelData);`

I have added the ONNX using the nuget package 
  <package id="Microsoft.ML.OnnxRuntime" version="1.15.1" targetFramework="net452" />
  <package id="Microsoft.ML.OnnxRuntime.Managed" version="1.15.1" targetFramework="net452" />

**Comment by xadupre** [comment]
Is it possible to know the error message?
